### PR TITLE
Menu for create poll from wall stream

### PR DESCRIPTION
**Pull request by yurabakhtin**
Issue: https://github.com/humhub/team_tasks/issues/239

### CODE REVIEW

**Review by luke- — CHANGES_REQUESTED** [review]
We need to refactor this module a bit more.
There should be no longer an "Polls" menu entry with Pollls only stream etc.

File: widgets/WallEntry.php [hunk 1]
```diff
@@ -15,11 +14,27 @@
...
     /**
      * @inheritDoc
      */
-    public $editRoute = "/polls/poll/edit";
+    public $editRoute = '/polls/poll/edit';
+
+    /**
+     * @inheritdoc
+     */
+    public $menuIcon = 'bar-chart';
```
[the remark this answers is not in the document]
Comment (yurabakhtin) [reply widgets/WallEntry.php]: @luke- Yes, it was already done here https://github.com/humhub/polls/pull/101/commits/7a4d7da87d9b3f9a49debda47fea9093753a0f5f.

**Comment by yurabakhtin** [comment]
@luke- 

> We need to refactor this module a bit more.
There should be no longer an "Polls" menu entry with Pollls only stream etc.

Ok, I will remove the "Polls" menu when I will find a solution for form on stream to make it work properly, because currently we have an issue with RichText on the loading form, it doesn't work for Post and Poll forms.

**Comment by yurabakhtin** [comment]
@luke- 

> We need to refactor this module a bit more.
There should be no longer an "Polls" menu entry with Pollls only stream etc.

Commit https://github.com/humhub/polls/pull/101/commits/66f7f8c571bc55fd89a7ff083c88d35f5eebd493.

**Review by luke- — APPROVED** [review]

**Review by luke- — APPROVED** [review]

**Pull request merged**